dlrover/python/master/node/job_manager.py

```diff
@@ -50,120 +50,129 @@ def __init__(
         self._error_monitor: ErrorMonitor = error_monitor
 
         self._job_nodes: Dict[str, Dict[int, Node]] = {}
+        self._nodes_required = (0, 0)
```

Comment:
Split `_nodes_required` to `_min_nodes` and `_max_nodes` which are more readable.

Reply:
just give a annotations here